### ISSUE TITLE
Conditional breakpoint

### ISSUE THREAD

**Issue by AbedKarmi** [x64dbg/x64dbg#1423]
It would be great if we can make a conditional breakpoint like OllyDBG
Also when press ctrl-G, would be great if we have a combo-box to choose from.

**Comment by mrexodia** [comment]
Already implemented and duplicate.

**Comment by AbedKarmi** [comment]
I could not break on button-click using x64dbg like in ollydbg, can we do the followings on x64?

- CONDITION :([[esp+4]+4] == WM_KEYDOWN) || ([[esp+4]+4] == WM_LBUTTONUP)
- Expression : [ESP+4]
- Decode : Pointer to MSGA
- Pause : OnCondition

**Comment by mrexodia** [comment]
You can indeed do that.

**Comment by mrexodia** [comment]
Click on edit breakpoint in the breakpoint view.

**Comment by AbedKarmi** [comment]
Yes, I know about this screen, but it is different from Ollydbg, how can I fill the above fields in this screen? i.e: where to put the Expression, and how to set the decoding type to MSGA ?

**Comment by mrexodia** [comment]
You can read the manual.

**Comment by mrexodia** [comment]
More specifically the whole introduction section. Without the basic
knowledge you won't get anywhere.
